ikarus/solver/nonlinearsolver/solverinfos.hh

```diff
@@ -23,9 +23,23 @@ struct NonLinearSolverInformation
    * \return `true` if the solver was successful, `false` otherwise.
    */
   explicit operator bool() const { return success; }
-  bool success{false};
-  double residualNorm{std::numeric_limits<double>::infinity()};
-  double correctionNorm{std::numeric_limits<double>::infinity()};
-  int iterations{-1};
+  bool success{false};                                            ///< Flag indicating the success of the solver.
+  double residualNorm{std::numeric_limits<double>::infinity()};   ///< Value of the residual norm.
+  double correctionNorm{std::numeric_limits<double>::infinity()}; ///< Value of the correction norm.
+  int iterations{-1}; ///< Total number of iterations performed by the non-linear solver.
+};
+
+/**
+ * \brief Information about the loggers of a non-linear solver.
+ *
+ * \details This structure holds information about the value of the load factor, the residual norm,
+ * the correction norm and the number of iterations performed.
+ */
+struct NonLinearSolverLoggingInformation
```

Comment:
Same discussion as above

Reply:
But maybe both things should be merged anyway. `NonLinearSolverInformation` and the logging information into one `State` or `info` struct.